lib/fathead/python_requests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+beautifulsoup4==4.5.1
+bs4==0.0.1
```

Comment:
for some reason this didn't seem to install bs4 correctly.

after running `pip install -r requirements.txt` I ran the parser and `bs4` was not found.

After running `pip instal bs4` everything worked as expected...

Not sure what causes that...